File: timer.cpp
// this program probably works only on Linux.

#include <iostream>
using namespace std;

int main()
{
  int timer, i, x;

  system("clear");
  cout << "Timer (seconds): ";
  cin >> timer;

  for (i = timer; i > 0; i--)
  {
    if (i >= 60)
    {
      x = i / 60;
      x = x + 1;
      if (x > 1)
      {
        system("clear");
        cout << x << " minutes (" << i << " seconds) remaining." << endl;
        system("sleep 1");
      }
      else if (x == 1)
      {
        system("clear");
        cout << x << " minute (" << i << " seconds) remaining." << endl;
        system("sleep 1");
        system("clear");
      }
    }
    else if (i < 60)
    {
      if (i > 1)
      {
        system("clear");
        cout << i << " seconds remaining." << endl;
        system("sleep 1");
      }
      else if (i == 1)
      {
        system("clear");
        cout << i << " second remaining." << endl;
        system("sleep 1");
        system("clear");
      }
    }
  }
  return 0;
}

File: timer.py
# this program probably works only on Linux.

import time
import os

os.system('clear')
timer = int(input('Timer (seconds): '))

for i in range(timer, 0, -1):
    if i >= 60:
        x = i // 60
        x = x + 1
        if x > 1:
            os.system('clear')
            output_string = str(str(x) + ' minutes (' + str(i) + ' seconds) remaining.')
            print(output_string)
            time.sleep(1)
        elif x == 1:
            os.system('clear')
            output_string = str(str(x) + ' minute (' + str(i) + ' seconds) remaining.')
            print(output_string)
            time.sleep(1)
    elif i < 60:
        if i > 1:
            os.system('clear')
            output_string = str(str(i) + ' seconds remaining.')
            print(output_string)
            time.sleep(1)
        elif i == 1:
            os.system('clear')
            output_string = str(str(i) + ' second remaining.')
            print(output_string)
            time.sleep(1)
os.system('clear')
